app/gui2/src/util/ast/extended.ts

```diff
@@ -130,6 +130,10 @@ export class AstExtended<T extends Tree | Token = Tree | Token, HasIdMap extends
   visitRecursive(visitor: (t: AstExtended<Tree | Token, HasIdMap>) => boolean) {
     visitGenerator(this.walkRecursive(), visitor)
   }
+
+  get parsedCode() {
```

Comment:
That function is misleading and should probably be removed. It will always return the whole input that was passed to the parser. You probably want to use `repr()` instead.

Reply:
But I _want_ to get the parsed code, so any inner Ast spans will refer to it. See the usage place in graphDatabase.ts, line 38